src/styles/blocks/catalog.scss

```diff
@@ -16,10 +16,17 @@
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
+    margin-bottom: $cardMargin;
```

Comment:
it is a great idea to create variables, but in my opinion variables for margins like that are redundant